email-templates/_shared/style.css

```diff
@@ -0,0 +1,349 @@
+    margin: 0 0 5px;
+    padding: 0px 45px 45px;
+    text-align: center;
+    word-wrap: break-word;
+}
+
+/***********
+ ** TILES **
+ ***********/
+
+.preview-tile-container {
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
+    border: 1px solid #DCE0E3;
+    height: 300px;
+    padding: 0px;
+    width: 300px;
+}
+
+.preview-tile-container.collabdoc {
+    background-position: top left;
+}
+
+.preview-tile-container-wide {
+    width: 520px;
+}
+
+.preview-tile-container-inner {
+    height: 300px;
+}
+
+.preview-tile-container .preview-tile-metadata {
+    background: #000000;
+    background-color: rgba(0, 0, 0, 0.8) !important;
+    padding: 7px;
+    width: 286px;
+}
+
+.preview-tile-metadata .preview-tile-metadata-inner {
+    width: 100%;
+}
+
+.preview-tile-container .preview-tile-metadata .preview-tile-metadata-title a {
+    color: #FFF;
+    display: block;
+    font-weight: 600;
+    line-height: 35px;
+    margin: 0;
+}
+
+.preview-tile-container .preview-tile-metadata .preview-tile-metadata-description {
+    color: #FFF;
+    padding-top: 2px;
+}
+
+.preview-asset-metadata {
+    color: #aeaaaa;
+    padding-right: 10px;
+    text-align: right;
+}
+
+.preview-asset-metadata img {
+    vertical-align: middle;
+    width: 24px;
+}
+
+
+/****************
+ ** THUMBNAILS **
+ ****************/
+
+.thumbnail-user img {
+    border-radius: 30px;
+    height: 40px;
+    width: 40px;
+}
+
+
+/****************
+ ** ACTIVITIES **
+ ****************/
+
+.activity-row > td {
+    background-color: #FFF;
+    border: 1px solid #EDEDEC;
+    margin: 10px auto;
+    padding: 50px;
+}
+
+.activity-row > td > table {
+    width: 100%;
+}
+
+.activity-row .activity-thumbnail-container {
+    float: left;
+}
+
+.activity-row .activity-summary-container {
+    color: #6A6B6A;
+    max-width: 620px;
+}
+
+.activity-row .activity-date {
+    color: #888;
+    margin-top: 5px;
+}
+
+.activity-row .activity-previews-container {
+    margin: 25px 0 0 -10px;
+}
+
+.activity-row .activity-comment-container {
+    margin-top: 25px;
+    min-height: 40px;
+}
+
+.activity-row .activity-comment-container.comment-level-1 {
+    margin-left: 40px;
+}
+
+.activity-row .activity-comment-container.comment-level-2 {
+    margin-left: 80px;
+}
+
+.activity-row .activity-comment-container .activity-comment-thumbnail-container {
+    float: left;
+}
+
+.activity-row .activity-comment-container .activity-comment {
+    padding-left: 52px;
+    max-width: 650px;
+}
+
+.activity-row .activity-comment-container.comment-level-1 .activity-comment {
+    max-width: 610px;
+}
+.activity-row .activity-comment-container.comment-level-2 .activity-comment {
+    max-width: 570px;
+}
+
+.activity-row .activity-comment-container .activity-comment .activity-comment-message {
+    margin-top: 7px;
+}
+
+/* Markdown formatting */
+
+.activity-row .activity-comment-container .activity-comment .activity-comment-message h1 {
```

Comment:
Comments and chat messages don't have any Markdown formatting. The only formatting that should be here is converting URLs to links.